tensorflow/compiler/xla/status_macros.h

```diff
@@ -196,6 +196,22 @@ class StatusAdaptorForMacros {
 #define TF_STATUS_MACROS_CONCAT_NAME(x, y) TF_STATUS_MACROS_CONCAT_IMPL(x, y)
 #define TF_STATUS_MACROS_CONCAT_IMPL(x, y) x##y
 
+#ifdef _MSC_VER
+
+#define TF_ASSIGN_OR_RETURN(lhs, rexpr)                                   \
```

Comment:
I think this is identical to TF_ASSIGN_OR_RETURN_IMPL_3 for the non-msc case.  Let's format the code to be identical, so that it's easier to see that they're the same.